WinUIGallery/ControlPages/TitleBarPage.xaml.cs

```diff
@@ -209,6 +209,24 @@ private void setTxtBoxAsPasthrough(FrameworkElement txtBoxNonClientArea)
             SetClickThruRegions(rectArr);
         }
 
+        private void TitleBarHeightComboBox_SelectionChanged(object sender, SelectionChangedEventArgs e)
+        {
+            var selectedHeight = ((ComboBoxItem)titlebarHeight.SelectedItem)?.Tag?.ToString();
+            var window = WindowHelper.GetWindowForElement(this);
+
+            if (selectedHeight != null && window.ExtendsContentIntoTitleBar)
+            {
+                window.AppWindow.TitleBar.PreferredHeightOption = selectedHeight switch
+                {
+                    "Collapsed" => TitleBarHeightOption.Collapsed,
+                    "Standard" => TitleBarHeightOption.Standard,
+                    "Tall" => TitleBarHeightOption.Tall,
+                    // This case will never be reached
+                    _ => throw new InvalidOperationException("Unreachable code reached in string pattern matching.")
```

Comment:
```suggestion
                    _ => TitleBarHeightOption.Standard
```
This is better than throwing an exception (even if the case is unreachable) because it provides a safer fallback, ensuring stability without unnecessary error handling.